src/app/notes/components/list-notes/list-notes.component.html

```diff
@@ -1,13 +1,15 @@
-<div *ngIf="currentNotes.length > 0; else noNotes" class="notes-container">
-  <mat-card *ngFor="let note of currentNotes" class="note-card">
-    <mat-card-header>
-      <mat-card-title>{{ note.title }}</mat-card-title>
-    </mat-card-header>
-    <mat-card-content>
-      <p>{{ note.description }}</p>
-    </mat-card-content>
-  </mat-card>
-</div>
+<ng-container *ngIf="notes$ | async as notes">
```

Comment:
We can use an AsyncPipe to subscribe to values emitted from an observable and bind them to the template. The pipe also handles unsubscribing before the component is destroyed which will prevent leaks.